src/utils.py

```diff
@@ -0,0 +1,20 @@
+# !/usr/bin/env python3
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Collection of helper methods for checking active connections between ZK and Kafka."""
```

Comment:
Maybe `for checking active connections` can be changed as the only one helper method here is not related to that purpose.